dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/generate-report-reconciled-email-variables.ts

```diff
@@ -0,0 +1,47 @@
+import { UtilisationReportEntity, formatDateForEmail, getFormattedReportPeriodWithShortMonth, ApiError } from '@ukef/dtfs2-common';
+import { getBankById } from '../../../../../repositories/banks-repo';
+import { FeeReportReconciledEmail } from '../../../../../types/utilisation-reports';
+import { NotFoundError, TransactionFailedError } from '../../../../../errors';
+
+/**
+ * generates variables for the record reconciled email
+ * gets the bank by the report bank id - errors if no bank founds
+ * generates the email variables including dates in the correct format
+ * @param report
+ * @returns emails and variables for the email
+ */
+export const generateReportReconciledEmailVariables = async (report: UtilisationReportEntity): Promise<FeeReportReconciledEmail> => {
+  try {
+    const bank = await getBankById(report.bankId);
+
+    if (!bank) {
+      console.error('Bank not found: %s', report.bankId);
+      throw new NotFoundError('Bank not found');
```

Comment:
Please can we include the bank id in the error message? Will be useful if we are ever investigating this error from logs